lib/query/builder.js

```diff
@@ -7,25 +7,20 @@ const { EventEmitter } = require('events');
 const Raw = require('../raw');
 const helpers = require('../helpers');
 const JoinClause = require('./joinclause');
-const {
-  assign,
-  clone,
-  each,
-  isBoolean,
-  isEmpty,
-  isFunction,
-  isNil,
-  isNumber,
-  isObject,
-  isString,
-  isUndefined,
-  tail,
-  toArray,
-  reject,
-  includes,
-  last,
-  isPlainObject,
-} = require('lodash');
+const assign = require('lodash/assign');
+const clone = require('lodash/clone');
+const each = require('lodash/each');
+const includes = require('lodash/includes');
```

Comment:
It seems that `includes` can be replaced with native implementation

Reply:
Knex supports running in the browser. I didn’t find a documentation of what versions are supported, so I sticked to ES5